### ISSUE TITLE
info.json should allow seeAlso for discovery of image content

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1507]
Use Case:  If an institution has images with metadata, but the metadata is not IIIF Presentation API, then their images cannot be discovered.  Instead, if the info.json allowed a reference via `seeAlso` to an external metadata format, then this could be used directly for the labels and descriptive metadata.  This doesn't muddy the water of presentation API's relationship with image, nor of what is in scope for info.json, it just allows a reference to an external document that may or may not be useful for a client to know about.

Towards discovery, this would decouple image and presentation for the purposes of finding descriptive information about the image, and allow additional notifications about image content becoming available separately from Manifests that reference that content. 

Proposal:

Import `seeAlso` from prezi3 into info.json in image3.

/ht Randy Stern

**Comment by tomcrane** [comment]
Should there be any particular guidance about when the metadata _is_ Presentation API?

"I am also to be found in this Manifest..."

**Comment by azaroth42** [comment]
I think we get this for free with `format` and `type`.

```
{
  "seeAlso": {
    "id": "https://example.com/iiif/1/manifest",
    "type": "Manifest"  // This would need importing...
    "format": "application/ld+json;profile=\"http://iiif.io/api/presentation/3/context.json\""
  }
}
```

**Comment by tomcrane** [comment]
In that scenario you could have both `seeAlso` and `partOf`

"There's some descriptive metadata about this image _here_, and it is also a service on one of the images in _this manifest_"

**Comment by azaroth42** [comment]
I wouldn't be averse to adding `partOf` as well.

**Comment by aisaac** [comment]
Meta-level comment: shouldn't this be discussed in the Discovery group?

**Comment by azaroth42** [comment]
It was yesterday, hence the issue :)

**Comment by jeffreycwitt** [comment]
As discussed on call from April 11, there is a concern about blurring the line between presentation info and image info. 

The addition of metadata and even manifest linking info seems practically useful, but I wonder if this actually information that more aptly applies to the canvas abstraction. 

As it is not a very common practice to publish de-reference-able canvases, to which information like "within/partOf" and "seeAlso" might more precisely apply, it is tempting to add this to the info.json. 

Nevertheless, this metadata and the fact that this image is found in a manifest more directly applies to the canvas. 

In other words, there is a reason the canvas abstraction was introduced, and it seems like we begin to confuse this by allowing metadata to be asserted in the info.json when it more accurately applies at the canvas level.

If we allow these additions, where does it stop? And what is the "principle" by which this limit is determined.

**Comment by azaroth42** [comment]
I agree with the concern in general, but I don't think that this falls squarely into that bucket. `seeAlso` is not specifically about presentation, instead it's a reference to another description that's related to the resource ... which could be a technical resource, or administrative, or rights related.  For example, it could be used to refer to FITS or Jhove derived preservation metadata about the content. 

I do see that `partOf` (previously called `within`) is stretching the boundaries though. My previous lack of aversion is swayed towards just `seeAlso`.  I also agree with the Canvas abstraction as a crucial part of the presentation model, especially for situations where there are multiple images inter-related via the Canvas. I don't think that `seeAlso` changes this directly, but I do see how it could be used to shortcut the relationship between image of object, physical object, metadata about physical object, presentation of image-and-metadata of physical object chain ... but at the moment the chain doesn't exist at all if the first link is the image.

Perhaps the solution to this is just the solution to #600, making the chain explicit?

**Comment by sdellis** [comment]
@azaroth42 just to clarify, by "solution to #600" you mean that "seeAlso" functionality would be implemented as a `service`?

**Comment by azaroth42** [comment]
No, that the way you find information about the image content to aid in discovery is not to associate the metadata about the depicted physical object (or other content) with the image, but instead to associate the info.json with the manifest, and the manifest with the discovery metadata.  So we don't short-circuit, keeping the distinction between presentation data, discovery data and technical image data clearer (per Jeff's point).

Basically it says that if you want to do discovery of IIIF Image API content, you need to also implement the Presentation API, as that's where discovery metadata will be referenced from.

**Comment by azaroth42** [comment]
Proposal:

Rename 5.7 Related Services to 5.7 Linking Properties
Add `seeAlso` for the technical metadata use case.
Add `partOf` for the linking to manifests use case, per #600.
